core/dbt/config/utils.py

```diff
@@ -62,6 +62,8 @@ def get_project_config(
             user_config = read_user_config(flags.PROFILES_DIR)
         # Update flags
         flags.set_from_args(args, user_config)
+        if cli_vars is None:
+            cli_vars = {}
```

Comment:
Is this just moving the code from `SecretRenderer` here?

This is the only part that scares me, insofar as it touches code that's more widely used than just `selectors.yml`. It's hard to imagine how this could cause issues, though.

Reply:
Got it! Thanks for the explanation